doc/patch-author-guide.md

```diff
@@ -808,6 +808,41 @@ This can be done by replacing any usages of `static_branch_likely()`,
 `static_branch_unlikely()`, `static_key_true()`, and `static_key_false()` with
 `static_key_enabled()` in the patch file.
 
+There is an example  of using `static_key_enabled()` to replace the static key in the patch file:
+This is an static key used in the patch file:
+```
+static inline bool memcg_kmem_enabled(void)
+{
+    return static_branch_unlikely(&memcg_kmem_enabled_key);
+}
+``` 
+    
+We can use `static_key_enabled()` to rewrite the macros `static_branch_likely` as follows:
+```
+#define static_branch_unlikely(x) unlikely(static_key_enabled(&(x)->key))
+```
+
+Using the following method to fix the static key error also work:
+```
+static inline bool memcg_kmem_enabled(void)
+{
+    return unlikely(&(&memcg_kmem_enabled_key)->key);
```

Comment:
Something looks missing or off here: is it just returning the address of the key member?  (Which would always decay to true?)

Reply:
I tried in my test case which is under kernel 4.9.
The function I rewrote it using `static_key_enabled(&memcg_kmem_enabled_key)`
+
+static inline bool memcg_kmem_enabled_self(void)
+{
+    return unlikely(static_key_enabled(&memcg_kmem_enabled_key));
+}
+
 However, I still got jump label error in my build.
The error out from create-diff-obejct is :
vmscan.o: changed function: shrink_slab.part.38
vmscan.o: changed function: shrink_slab
vmscan.o: Found a jump label at shrink_slab.part.38()+0x12e, using key memcg_kmem_enabled_key.
/usr/local/libexec/kpatch/create-diff-object: ERROR: vmscan.o: kpatch_regenerate_special_section: 1654: Found 1 jump label(s) in the patched code. Jump labels aren't currently supported. Use static_key_enabled() instead.

Actually, I dont know why this method not work.